default/stringtables/en.txt

```diff
@@ -5032,7 +5032,14 @@ SITREP_SNOOZE_CLEAR_INDEFINITE
 '''  Show all SitReps indefinitely dismissed'''
```

Comment:
If it is ok, I'd like to address clarification within #1019, so one PR is not dependent on the other.
I've reverted this fix here, but am happy to reapply if needed or the other is rejected.